plugin.php

```diff
@@ -295,6 +295,18 @@ function rest_validate_request_arg( $value, $request, $param ) {
 			}
 		}
 
+		if ( 'array' === $args['type'] ) {
+			// A comma-separated array of IDs could be acceptable instead of an array.
```

Comment:
This isn't the right abstraction.

JSON Schema permits us to specify two different types for a given property. In the case of anything using `wp_parse_id_list()`, we need to specify `type => [ 'string', 'array' ]`, and then properly validate an array of types.